src/main/java/seedu/modtracker/ModuleList.java

```diff
@@ -48,19 +109,35 @@ public void addMod(String input, boolean toPrint, Storage storage) {
      * @param storage storage object where data is stored
      */
     public void addExp(String input, boolean toPrint, Storage storage) {
-        String[] modInfo = input.split(" ", 3);
-        modInfo[1] = modInfo[1].toUpperCase();
-        Module currentMod = new Module(modInfo[1], modInfo[2]);
-        if (!checkModuleIfExist(modInfo[1])) {
-            modList.add(currentMod);
-        } else {
-            int index = modList.indexOf(currentMod);
+        try {
+            String[] modInfo = input.split(" ", 3);
+            modInfo[1] = modInfo[1].toUpperCase();
```

Comment:
Maybe can label as a comment what the various index of the array modInfo refers to